editor/editor_inspector.cpp

```diff
@@ -1178,7 +1148,7 @@ void EditorInspectorCategory::_notification(int p_what) {
 }
 
 Control *EditorInspectorCategory::make_custom_tooltip(const String &p_text) const {
-	return make_help_bit(TTR("Class:"), p_text, String(), Color());
+	return memnew(EditorHelpTooltip(p_text));
```

Comment:
This needs treatment similar to `ConnectionsDockTree::make_custom_tooltip` to address @akien-mga's report. Seems like in https://github.com/godotengine/godot/pull/81221 @dalexeev added the hint to regular categories, but for custom categories it was only a name, and `make_help_bit` would ignore those without raising an error.

Since the tooltip control raises an error now, the check must be added here.